src/main/java/nl/mpcjanssen/simpletask/util/Config.kt

```diff
@@ -142,20 +142,8 @@ object Config : SharedPreferences.OnSharedPreferenceChangeListener {
         return prefs.getBoolean(getString(R.string.append_tasks_at_end), true)
     }
 
-    private fun themeStringToId(theme: String): Int {
-        when (theme) {
-            "dark" -> return R.style.AppTheme
-            "black" -> return R.style.AppTheme_Black
-            "light_darkactionbar" -> return R.style.AppTheme_Light_DarkActionBar
-        }
-        return R.style.AppTheme_Light_DarkActionBar
-
-    }
-
-    val activeTheme: Int
-        get() {
-            return themeStringToId(activeThemeString)
-        }
+    // Takes an argument f, an expression that maps theme strings to IDs
+    val activeTheme = { f: (String) -> Int -> f(activeThemeString) }
```

Comment:
Nested lambda functions : The syntax is terrible but with the comment it's less terrible.

I didn't strictly speaking need to do this -- it's a mild performance improvement but really, we're not optimizing right now -- but it was fun and helped me learn how kotlin does lambdas. If it's not clear enough, you can change it to
```
fun activeTheme(f: (String) -> Int): Int {
    return f(activeThemeString)
}
```